src/packages/cli/index.ts

```diff
@@ -1,3 +1,3 @@
 import Ganache from "@ganache/core";
-export { ServerOptions, ProviderOptions } from "@ganache/core";
+export { ServerOptions, ProviderOptions, Status } from "@ganache/core";
```

Comment:
Exporting as I wanted to use the server's actual `Status` flags in cli instead of hardcoding the status values there.